src/service/viewer-impl.js

```diff
@@ -835,6 +835,34 @@ export class Viewer {
   }
 
   /**
+   *
+   * @return {!Promise}
+   */
+  replaceShareTrackingFragment(incomingFragment, outgoingFragment) {
+    if (!this.isEmbedded_) {
```

Comment:
I don't think `isEmbedded` is the right check here, but maybe it is - please check. Basically, `isEmbedded = isIframed || isInWebView`. You might have to do different things based on whether the page is iframed or is in webview.